.github/workflows/on-tag.yml

```diff
@@ -0,0 +1,47 @@
+name: Publish tag docs to gh-pages
+
+on:
+  push:
+    tags: v/[1-9]+.[0-9]+.[0-9]+
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout sources
+      uses: actions/checkout@v2
+
+    - name: Use Node.js 14.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 14.x
+
+    - name: Cache Node.js modules 💾
+      uses: actions/cache@v2
+      with:
+        path: ~/.npm
+        key: ${{ runner.OS }}-node-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.OS }}-node-
+          ${{ runner.OS }}-
+
+    - name: Install dependencies ⏬
+      run: npm install
+
+    - name: Cleanup and prepare pages
+      run: |
+        rm -rf ${GITHUB_REF##*/}
```

Comment:
Where does `GITHUB_REF` come from?

Reply:
That's automatically set by the system (and actually an alias/shortcut for `${{ github.ref }}`)